testutil/promrated/promrated/main.go

```diff
@@ -42,7 +42,7 @@ func bindPromratedFlag(flags *pflag.FlagSet, config *promrated.Config) {
 	flags.StringVar(&config.RatedEndpoint, "rated-endpoint", "https://api.rated.network", "Rated API endpoint to poll for validator metrics.")
 	flags.StringVar(&config.RatedAuth, "rated-auth-token", "token", "[REQUIRED] Token for Rated API.")
 	flags.StringVar(&config.MonitoringAddr, "monitoring-address", "127.0.0.1:9100", "Listening address (ip and port) for the prometheus monitoring http server.")
-	flags.StringVar(&config.PromEndpoint, "prom-endpoint", "https://vm.monitoring.gcp.obol.tech/query", "Endpoint for VMetrics Prometheus API.")
 	flags.StringVar(&config.PromAuth, "prom-auth-token", "token", "[REQUIRED] Token for VMetrics Promtetheus API.")
 	flags.StringSliceVar(&config.Networks, "networks", nil, "Comma separated list of one or networks to monitor.")
+	flags.StringSliceVar(&config.Networks, "node-operators", nil, "Comma separated list of one or node operators to monitor.")
```

Comment:
Copy-paste error!

Reply:
Also, do we need any validation for these? If the node operators list is not provided then the labels won't be populated?

Reply:
that would be the intended behaviour, I think that this would be OK unless you have a strong objection?